README.md

```diff
@@ -1,76 +1,39 @@
 # CUDA-Python
 
-CUDA Python is a standard set of low-level interfaces, providing full coverage of and access to the CUDA host APIs from Python. Checkout the [Overview](https://nvidia.github.io/cuda-python/overview.html) for the workflow and performance results.
+CUDA Python is the home for accessing NVIDIA’s CUDA platform from Python. It consists of multiple components:
 
-## Installing
+* [cuda.core](https://nvidia.github.io/cuda-python/cuda-core/latest): Pythonic access to CUDA Runtime and other core functionalities
+* [cuda.bindings](https://nvidia.github.io/cuda-python/cuda-bindings/latest): Low-level Python bindings to CUDA C APIs
+* [cuda.cooperative](https://nvidia.github.io/cccl/cuda_cooperative/): Pythonic exposure of CUB cooperative algorithms
+* [cuda.parallel](https://nvidia.github.io/cccl/cuda_parallel/): Pythonic exposure of Thrust parallel algorithms
 
-CUDA Python can be installed from:
+For access to NVIDIA Math Libraries, please refer to [nvmath-python](https://docs.nvidia.com/cuda/nvmath-python/latest).
 
-* PYPI
-* Conda (nvidia channel)
-* Source builds
+CUDA Python is currently undergoing an overhaul to improve existing and bring up new components. All of the previously available functionalities from the cuda-python package will continue to be available, please refer to the [cuda.bindings](https://nvidia.github.io/cuda-python/cuda-bindings/latest) documentation for installation guide and further detail.
 
-There're differences in each of these options that are described further in [Installation](https://nvidia.github.io/cuda-python/install.html) documentation. Each package will guarantee minor version compatibility.
+## CUDA-Python as a metapackage
 
-## Runtime Dependencies
+CUDA-Python is structured to become a metapackage that contains a collection of subpackages. Each subpackage is versioned independently, allowing installation of each component as needed.
 
-CUDA Python is supported on all platforms that CUDA is supported. Specific dependencies are as follows:
+### Subpackage: `cuda.core`
 
-* Driver: Linux (450.80.02 or later) Windows (456.38 or later)
-* CUDA Toolkit 12.0 to 12.6
+The `cuda.core` package offers idiomatic, pythonic access to CUDA Runtime and other functionalities.
 
-Only the NVRTC redistributable component is required from the CUDA Toolkit. [CUDA Toolkit Documentation](https://docs.nvidia.com/cuda/index.html) Installation Guides can be used for guidance. Note that the NVRTC component in the Toolkit can be obtained via PYPI, Conda or Local Installer.
+The goals are to
 
-### Supported Python Versions
+1. provide **idiomatic (“pythonic”)** access to CUDA Driver/Runtime
```

Comment:
subjective nit, capitalization of the first work in each numbered point.

Reply:
Done